source/simulator/solver_schemes.cc

```diff
@@ -266,7 +266,7 @@ namespace aspect
         // and will therefore be updated as well.
         const unsigned int fluid_velocity_block = introspection.variable("fluid velocity").block_index;
         const unsigned int fluid_pressure_block = introspection.variable("fluid pressure").block_index;
-        current_linearization_point.block(fluid_velocity_block) = solution(fluid_velocity_block);
+        current_linearization_point.block(fluid_velocity_block) = solution.block(fluid_velocity_block);
```

Comment:
Did we just accidentally copy out a single element here and assign it to a whole vector? :-)

Reply:
Yes, and I just understood that the result is luckily currently not used (the fluid velocity is not one of the primary solution variables, it is more a postprocessed field that is computed from the two pressures and the solid velocity). Thus, all models that ran since #1850 actually computed the correct results (everything beforehand was fine). Only parallel models after #1850 would crash, because they accessed this one element.